docs/tutorial.md

```diff
@@ -71,20 +71,50 @@ Variable are used to save data and use it later in the code.
 
 Now save the file so we can run the code. Open a terminal again (or a new terminal in VSCode) and navigate to the same folder the file is saved in. This can be done using the `cd` command, for example: `cd Documents\javascript`  
 Once your terminal is in the same folder as your Javascript file, you can run `node filename.js`  
-If you have done everything correctly, you should see nothing.
+If you have done everything correctly, you should see nothing.  
+In the next chapter we will show you how you can 'print' things to the terminal.
+
+In general, it is good practice to use the `const` keyword instead of the `let` keyword when defining a variable. A variable defined with `const` can't be modified later and is thus constant.  
```

Comment:
> and is thus constant

i think that `and thus is a constant` would be more correct

Reply:
That does sound nicer.